### PR TITLE
CLOUD-41495 Kerberos support

### PR DESCRIPTION
**Pull request by mhmxs**
@keyki plz review

### CODE REVIEW

**Comment by doktoric** [comment]
database migration scripts are missing

File: core/src/main/java/com/sequenceiq/cloudbreak/EnvironmentVariableConfig.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
     public static final String CB_DOCKER_CONTAINER_REGISTRATOR = "sequenceiq/registrator:v5.2";
     public static final String CB_DOCKER_CONTAINER_DOCKER_CONSUL_WATCH_PLUGN = "sequenceiq/docker-consul-watch-plugn:2.0.0-consul";
     public static final String CB_DOCKER_CONTAINER_AMBARI_DB = "postgres:9.4.1";
+    public static final String CB_DOCKER_CONTAINER_KERBEROS = "sequenceiq/kerberos";
```
Comment (doktoric) [inline core/src/main/java/com/sequenceiq/cloudbreak/EnvironmentVariableConfig.java]: dont use the latest. please use a specific version

**Comment by doktoric** [comment]
If you add a new container to the image then you have to rebuild our cloud images

**Comment by keyki** [comment]
Please create the sql migration script if there are changes.

**Comment by mhmxs** [comment]
There was no database schema change.

**Pull request merged**